README_zh.md

```diff
@@ -76,7 +76,7 @@ $ make docker-build
 ```
 
 ### 部署
-请参考 [docs/installer/install_zh.md](docs/installer/installation_zh.md)
+请参考 [docs/installation/installation_zh.md](docs/installation/installation_zh.md)
```

Comment:
Can we move all the zh doc to `/docs/docs_zh`?

Reply:
New a issue to resolve it when write docs in this sprint, this issue close now